### ISSUE TITLE
States don't update, not possible to fill in a foreign address with last release

### ISSUE THREAD

**Issue by myriamboure** [openfoodfoundation/openfoodnetwork#1520]
On staging in France we have the v.1.8.9 version, and we have the Italian local states imported. I want to create a new enterprise, I can select "italy' in country menu, but the states don't update, so I can't select a state, so the address can't be found ("AIN" in the example is a French state).

![screenshot from 2017-03-30 17-41-15](https://cloud.githubusercontent.com/assets/8135376/24659397/dccb32a4-194c-11e7-928f-e7be206e07ee.png)

Ping @sigmundpetersen who has a connected issue.

**Comment by myriamboure** [comment]
I label this as critical as we have a spanish user, so we can't  upgrade while this is not solved... @pacodelaluna do you think you can have a look at what happens?

**Comment by daniellemoorhead** [comment]
Did you mean release 1.8.8 @myriamboure?

**Comment by sigmundpetersen** [comment]
Yes I think so @daniellemoorhead , this failed for Scandinavia when updating from 1.8.5 to 1.8.7 and still din't work when updating to 1.8.8. We can't even choose country as France say they can. Our connected discussion here https://community.openfoodnetwork.org/t/choosing-country-doesnt-work-after-upgrade/798/5

**Comment by oeoeaio** [comment]
@sigmundpetersen I can't seem to get this to work on any recent version. I have tried v1.8.5, v1.8.4 and v1.8.0. Are you sure it ever worked? I can't see any code that would be doing the job of updating the list of states...

**Comment by sigmundpetersen** [comment]
@oeoeaio as you see on the map https://www.openfoodnetwork.no/map we have managed to configure an enterprise to a Swedish address. The address stays correct in the background it seems as you can see on this picture (Uppsala is in Sweden), but now we're only able to choose Norway as country.

![address](https://cloud.githubusercontent.com/assets/1451633/24789974/8e3389b8-1b75-11e7-8747-733b668bfa33.png)

Trying to create a new enterprise it's not possible to choose any other country either.

**Comment by myriamboure** [comment]
When reading @oeoeaio comment I tried something else... I changed the country in the address of the hub, then save. An error message appears:
![screenshot from 2017-04-19 09-49-45](https://cloud.githubusercontent.com/assets/8135376/25168949/99ab1c56-24e5-11e7-91b5-ea7d7e9b3085.png)
BUT when I click again on "address" I see that now the right list of states is displayed according to the new country set up. So I'll close that issue and open a new one to update dynamically the list of state so that we improve a little bit the UX... as it's not very intuitive ;-) But it doesn't prevent us from working. @sigmundpetersen I encourage you to open another github issue if your issue is different. That works i France to set up hubs in Spain so there is no reason why it couldn't work in Norway with Sweden...
